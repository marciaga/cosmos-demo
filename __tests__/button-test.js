import React from 'react';
import renderer from 'react-test-renderer';
import { Loader } from 'react-cosmos-loader';
import Button from '../src/atomic/button';
import fixture from '../src/atomic/button/__fixtures__/default';

test('it renders a button component', () => {
  const tree = renderer.create(<Loader component={Button} fixture={fixture} />).toJSON();
  expect(tree).toMatchSnapshot();
});
