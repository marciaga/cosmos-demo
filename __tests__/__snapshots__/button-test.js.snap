// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a button component 1`] = `
<button
  foo={false}
  onClick={[Function]}
  type="submit"
>
  Click me
</button>
`;
